widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -173,15 +173,27 @@ export const Chains = styled(Collapsible.Trigger, {
   },
   '& div:nth-child(1)': { display: 'flex' },
 });
+export const Row = styled('div', {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'center',
+  '.chainImage': {
```

Comment:
To prevent a clash in class names with the application utilizing the widget, kindly generate an additional element or check https://github.com/rango-exchange/rango-client/pull/471

Reply:
fixed